starter-code/apple-pie-starter-code/index.html

```diff
@@ -2,34 +2,55 @@
 <html lang="en">
```

Comment:
No has usado el plugin built-in que tiene VSCode, emmet, para generar correctamente los metadatos del documento HTML. Prueba con con un fichero vacío, lo guardas como test.html y escribe html:5 y le das al tabulador. Ya me cuentas